ambari-server/src/main/java/org/apache/ambari/server/upgrade/UpgradeCatalog270.java

```diff
@@ -1079,6 +1081,17 @@ private boolean updateWebHCatHostsInHadoopProxyuserHttpHostsForHive(KerberosDesc
             updated = true;
           }
         }
+        final KerberosConfigurationDescriptor webhcatSiteConfiguration = webhcatServer.getConfiguration(CONFIGURATION_WEBHCAT_SITE);
```

Comment:
You might want to rename the container method - `updateWebHCatHostsInHadoopProxyuserHttpHostsForHive` - to something like `updateWebHCatHostsInKerberosDescriptors` or something like that.  Or move the new block of code to a new method.  The current way is not very intuitive.

Reply:
Modified the function name, also added a short description of it in a comment.

Reply:
thanks